gradle.properties

```diff
@@ -7,6 +7,8 @@
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
 org.gradle.jvmargs=-Xmx2048m
```

Comment:
I think this should be set to 4096

Reply:
I increased the Docker Engine, vmargs and JAVA_OPTS all to 6G, but still hitting the same "Lint infrastructure error" on: 
```
In configuration ':app:armDebugAndroidTestCompileClasspath': substituting megazord module 'org.mozilla.appservices:fenix-megazord:0.28.1' for component module 'org.mozilla.appservices:places:0.28.1'
In configuration ':app:armDebugAndroidTestCompileClasspath': substituting megazord module 'org.mozilla.appservices:fenix-megazord:0.28.1' for component module 'org.mozilla.appservices:fxaclient:0.28.1'
In configuration ':app:armDebugAndroidTestCompileClasspath': substituting megazord module 'org.mozilla.appservices:fenix-megazord:0.28.1' for component module 'org.mozilla.appservices:rustlog:0.28.1'
Ran lint on variant aarch64Raptor: 276 issues found
Ran lint on variant armDebug: 289 issues found
Ran lint on variant aarch64Nightly: 276 issues found
Ran lint on variant x86Nightly: 276 issues found
Ran lint on variant x86Raptor: 276 issues found
Ran lint on variant armBeta: 149 issues found
Ran lint on variant x86Debug: 289 issues found
Ran lint on variant x86Beta: 276 issues found
Ran lint on variant aarch64Debug: 289 issues found
Ran lint on variant armNightly: 276 issues found
Ran lint on variant aarch64Beta: 276 issues found
Ran lint on variant armRaptor: 276 issues found

> Task :app:lint FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:lint'.
> Lint infrastructure error
  Caused by: java.lang.reflect.InvocationTargetException
  	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMeth
```